README.md

```diff
@@ -15,39 +15,23 @@ Before you begin, make sure you have the required environment:
 
 ## Basic installation
 
-The chart uses `nodeAffinity` for mounting Persistent Volume of type `local`.
-This also allows the user to specify which node will host the WProofreader Server 
-on a cluster (even a single-node one).
-
-To assign this role to a node, one has to attach a label to it. It can be whatever you want it to be,
-e.g. `proofreader.your-company.com/app`:
-```shell
-kubectl label node <name-of-the-node> proofreader.company-domain.com/app=
-```
-Note that `=` is required, but the value after it is not important (empty in this example).
-
-Keep in mind that your custom label has to be either updated in `values.yaml`
-(`affinityLabel` key, recommended), or passed to `helm` calls using 
-`--set affinityLabel=proofreader.company-domain.com/app`.
-
-Now, the chart can be installed the usual way using all the defaults:
+The Chart can be installed the usual way using all the defaults:
 ```shell
 git clone https://github.com/WebSpellChecker/wproofreader-helm.git
 cd wproofreader-helm
-helm install --create-namespace --namespace wsc wsc-app-5-x-x wproofreader --set affinityLabel=proofreader.company-domain.com/app 
```

Comment:
do you think there is no sense to keep this naming/versioning as a default?

Reply:
In my opinion, it is better to have it the way it is. 
The Helm release name is the part of naming convention here and the resources that will be created will contain it in their names. 
So having a version there will confuse more when using `helm update` command or `--set image.tag`, for example. 
But depending on deployment strategy it is also possible to have a version there.